packages/flutter/test/material/theme_defaults_test.dart

```diff
@@ -143,7 +143,7 @@ void main() {
 
       final RawMaterialButton raw = tester.widget<RawMaterialButton>(find.byType(RawMaterialButton));
       expect(raw.textStyle.color, const Color(0xdd000000));
-      expect(raw.fillColor, const Color(0x00ffffff));
+      expect(raw.fillColor, const Color(0x00fafafa));
```

Comment:
Should there also be an `expect` for raw.canvasColor to be the same color here?

Reply:
This test is just checking the properties of the RawMaterialButton widget created for an OutlineButton. RawMaterialButton doesn't have a canvasColor property. 

RawMaterialButton is the widget most of the button classes build because it doesn't depend on any theme values.